src/components/VersionRestartModal/VersionRestartModal.tsx

```diff
@@ -139,43 +136,117 @@ const VersionRestartModal: React.FC<Props> = ({
       ]}
       data-cy="version-restart-modal"
     >
+      <VersionTasks
+        version={version}
+        selectedTasks={selectedTasks}
+        setBaseStatusFilterTerm={setVersionBaseStatus(version?.id)}
+        setPatchStatusFilterTerm={setVersionStatus(version?.id)}
+        toggleSelectedTask={toggleSelectedTask}
+        baseStatusFilterTerm={baseStatusFilterTerm[version?.id]}
+        patchStatusFilterTerm={patchStatusFilterTerm[version?.id]}
+      />
+
+      {childVersions && (
+        <div data-cy="select-downstream">
+          <ConfirmationMessage weight="medium" data-cy="confirmation-message">
+            Downstream Tasks
+          </ConfirmationMessage>
+          {childVersions?.map((v) => (
+            <div key={v?.id}>
```

Comment:
This wrapping div doesn't seem necessary?

Reply:
it was erring without